tests/test_octodns_provider_splityaml.py

```diff
@@ -0,0 +1,212 @@
+#
```

Comment:
Would prefer to either include these tests in `tests/test_octodns_provider_yaml.py` or to spilt the provider out into two modules (`octodns/provider/split_yaml.py`) so that the module and tests match up grouping-wise.

Reply:
Rolled the tests into `tests/test_octodns_provider_yaml.py`, because I think specifying a provider class name `octodns.provider.yaml.SplitYamlProvider` is a bit cleaner.